weblab/templates/registration/register.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  <h1>Register for this Functional Curation system</h1>
+
+  <p>
+    Only users with an account are able to upload their own models and/or
+    protocols for analysis. Once you have registered, you will need to wait
+    for an admin to approve the account before you are able to upload files.
+  </p>
+
+  <form action="" method="POST">
+    {% csrf_token %}
+    {{ form.as_p }}
```

Comment:
When you register in Firefox, the password details it offers to save have your institution as the username! Is it possible to tell Firefox that the email address is the username instead?

Reply:
Sure... unfortunately firefox (and possibly other browsers?) uses the field before the password field as username if it can't find one called username. I've changed the order to accommodate this.